designcompose/src/main/java/com/android/designcompose/FrameRender.kt

```diff
@@ -544,8 +544,8 @@ internal fun ContentDrawScope.render(
         vectorSize.ifPresent {
             val sizeList = vectorSize.get()
             if (sizeList.size == 2) {
-                val vecWidth = sizeList[0] * vectorScaleX
-                val vecHeight = sizeList[1] * vectorScaleY
+                val vecWidth = sizeList[0] * vectorScaleX * density
+                val vecHeight = sizeList[1] * vectorScaleY * density
```

Comment:
Good catch, thanks. In the new layout system I'm working on, the size of the draw context is determined by layout, and the size to draw is determined by the size of the vector from Figma. I'll have to think about if this will need to change again in the new system. I'll test this change out on Tuesday.

Reply:
Yeah, I think it will; I'm making an emulator with dpi 160 to test out the layout change right now...